fbw-common/src/systems/instruments/src/EFB/AircraftContext.ts

```diff
@@ -24,7 +67,41 @@ export const AircraftContext = createContext<AircraftEfbContext>({
     takeoff: null,
     landing: null,
   },
+  pushbackPage: {
+    turnIndicatorTuningDefault: 0,
+  },
   settingsPages: {
+    audio: {
+      announcements: false,
+      boardingMusic: false,
+      engineVolume: false,
+      masterVolume: false,
+      windVolume: false,
+      ptuCockpit: false,
+      paxAmbience: false,
+    },
+    pinProgram: {
+      latLonExtend: false,
+      paxSign: false,
+      rmpVhfSpacing: false,
+      satcom: false,
```

Comment:
FIXME: just inject the aircraft options page from the aircraft context (or plugin in flypadOSv4).

Reply:
Yes, will add the comment